deps/spidershim/spidermonkey-patches/02-NewArrayBufferWithExternalContents.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/deps/spidershim/spidermonkey/js/src/jsapi.h b/deps/spidershim/spidermonkey/js/src/jsapi.h
```

Comment:
On Tue, May 10, 2016 at 09:57:11AM -0700, Ehsan Akhgari wrote:
> In deps/spidershim/spidermonkey-patches/
> 02-NewArrayBufferWithExternalContents.patch:
> 
> > @@ -0,0 +1,39 @@
> > +diff --git a/deps/spidershim/spidermonkey/js/src/jsapi.h b/deps/spidershim/spidermonkey/js/src/jsapi.h
> 
> Can you please upstream this patch so that we don't have to maintain it in
> spidershim? (Obviously landing it downstream in the mean time is fine!)

I was already doing that